iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/x509/user.x509.component.js

```diff
@@ -290,7 +309,8 @@
                     cert: undefined,
                     successHandler: function () {
                         return self.handleSuccess;
-                    }
+                    },
+                    certificationAuthorities: () => TrustsService.getTrusts()
```

Comment:
We need to define the certificationAuthorities also in the resolve block of `self.openRemoveCertificateDialog` method, otherwise we get an angularjs error when we click on the `Remove` button